tests/integ/scala/test_function_suite.py

```diff
@@ -2851,6 +2851,103 @@ def test_array_remove(session):
         sort=False,
     )
 
+    actual = session.createDataFrame([([1, 2, 4, 4, 3],), ([],)], ["data"])
+    actual = actual.select(array_remove(actual.data, 4))
+    expected = session.createDataFrame(
+        [
+            Row("[\n  1,\n  2,\n  3\n]"),
+            Row("[]"),
+        ],
+        ["data"],
+    )
```

Comment:
not: creating a dataframe here (and other updates below) is not necessary

Reply:
I delete the dataframe.